README.md

```diff
@@ -2,18 +2,18 @@
 
 List of tech companies (or teams) that are based or working in Nepal. :nepal:
 
-|     |     |     |     |     |     |     |     |     |
-|:-:  |:-:  |:-:  |:-:  |:-:  |:-:  |:-:  |:-:  |:-:  |
+|                 |                 |                 |                 |                 |                 |                 |                 |                 |
```

Comment:
@azens1995 Thank you for the PR. Can you revert this change? I think you are using a code formatted like prettier?

Reply:
Yes, @mesaugat dai. These are changes by the prettier. OK. I will revert those changes back.